src/lib/trajectory/index.ts

```
@@ -0,0 +1,1007 @@
+    const energy_fields = [
+      `energy`,
+      `energy_per_atom`,
+      `potential_energy`,
+      `kinetic_energy`,
+      `total_energy`,
+    ]
+
+    for (const field of energy_fields) {
+      if (
+        field in frame.metadata &&
+        typeof frame.metadata[field] === `number`
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)